index.js

```diff
@@ -113,7 +113,7 @@ const ScrollableTabView = React.createClass({
             return <View
               key={child.props.tabLabel + '_' + idx}
               style={{width: this.state.containerWidth, }}>
-              {child}
+              { React.cloneElement(child, { isActiveTab:(idx === this.state.currentPage)}) }
```

Comment:
React.cloneElement is already used in this way in this same component, see:
https://github.com/brentvatne/react-native-scrollable-tab-view/blob/master/index.js#L76

In my app, no other existing props were disrupted when adding this one. It was actually my heavy use of this component that instilled my confidence in using cloneElement as a solid technique.